local_minio/minio_install.sh

```diff
@@ -1,69 +1,76 @@
 #!/bin/bash
+source "utils/parameters.sh"
 
+# IPS_ARR=(
 
-IPS_ARR=(
-    192.168.89.6
-    192.168.89.13
-    192.168.89.20
-    192.168.89.5
-    192.168.89.17
-    192.168.89.10
-    192.168.88.253
-    192.168.89.21
-    192.168.89.0
-    192.168.89.22
-    192.168.89.4
-    192.168.89.8
-    192.168.89.14
-    192.168.89.9
-    192.168.89.2
-    192.168.88.254
-    192.168.89.1
-    192.168.89.24
-    192.168.89.29
-    192.168.89.103
-    192.168.88.252
-    192.168.89.16
-    192.168.89.11
-    192.168.89.18
-    192.168.89.7
-    192.168.89.3
-    192.168.89.19
-    192.168.89.15
-    192.168.89.64
-    192.168.89.65
-    192.168.89.104
-    192.168.89.91
-)
-ACCESS_KEY=access_key
-SECRET_KEY=secret_key
-GROUPID=jimmy-minio-server
+GROUPID=minio-server
```

Comment:
Let's get this from the parameters (it is a default parameter that the backend always sends)